mlxtend/preprocessing/onehot.py

```diff
@@ -119,12 +120,19 @@ def transform(self, X):
            ['Milk', 'Beer', 'Rice'],
            ['Milk', 'Beer'],
            ['Apple', 'Bananas']]
+
+        sparse: bool
+          If True, transform will return Compressed Sparse Row matrix
+          instead of the regular one.
+
         Returns
         ------------
-        onehot : NumPy array [n_transactions, n_unique_items]
-           The NumPy one-hot encoded integer array of the input transactions,
+        onehot : NumPy array [n_transactions, n_unique_items] If not sparse
```

Comment:
I would suggest changing 

"If not sparse" 

to 

"if `sparse=False` (default)   
and  SciPy Compressed Sparse Row (CSR) matrix, otherwise